cumulus-client/build.gradle

```diff
@@ -2,7 +2,13 @@ dependencies {
     implementation(project(":cumulus-model"))
     implementation("mil.army.usace.hec:cwms-http-client:1.0-SNAPSHOT")
 
+    testImplementation("mil.army.usace.hec:cwms-http-client:1.0-SNAPSHOT")
```

Comment:
isnt this added right above?

Reply:
Yeah, added that in when i was testing the radar client update I made. I'll remove it.